composer.json

```diff
@@ -26,7 +26,8 @@
         "symfony/phpunit-bridge": "^4.0",
         "twig/twig": "^1.32 || ^2.4",
         "symfony/monolog-bundle": "^3.2",
-        "friendsofphp/php-cs-fixer": "^2.11"
+        "friendsofphp/php-cs-fixer": "^2.11",
+        "phpunit/phpunit": "^7.5"
```

Comment:
Not related to that PR. is it?

Reply:
@jderusse Is explained in the PR Message:

> This PR also causes adds phpunit/phpunit ^7.5 as an explicit dev dependency, since people might use a global phpunit ^8 dependency and this version is incompatible with the tests (but fixing the tests themselves would be out of the scope of this PR).

Reply:
can you please create a dedicated PR for that change?

Reply:
@jderusse here you go: https://github.com/ekino/EkinoNewRelicBundle/pull/227

Reply:
Should we have another PR for fixing the CI errors as well?

Reply:
did it in #226